bundle/OpenGraph/Handler/ImageFallback.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\SiteBundle\OpenGraph\Handler;
+
+use Netgen\Bundle\OpenGraphBundle\Handler\ContentAware;
+use eZ\Publish\Core\Base\Exceptions\InvalidArgumentException;
+use eZ\Publish\API\Repository\Values\Content\Field;
+use Netgen\Bundle\OpenGraphBundle\Handler\HandlerInterface;
+use Netgen\Bundle\OpenGraphBundle\Handler\FieldType\Handler;
+use eZ\Publish\Core\FieldType\Image\Value as ImageValue;
+use Netgen\Bundle\RemoteMediaBundle\Core\FieldType\RemoteMedia\Value as RemoteImageValue;
+use eZ\Publish\Core\Helper\FieldHelper;
+use eZ\Publish\Core\Helper\TranslationHelper;
+use Netgen\Bundle\OpenGraphBundle\Exception\FieldEmptyException;
+use Netgen\Bundle\OpenGraphBundle\Handler\FieldType\Image as ImageHandler;
+use Netgen\Bundle\RemoteMediaBundle\OpenGraph\Handler\RemoteMediaHandler;
+
+final class ImageFallback extends Handler implements HandlerInterface, ContentAware
```

Comment:
What's the usecase for this open graph handler?

Reply:
To make it easier to switch between ezimage and ngremotemedia fields. If we do not have this, we would need different configuration for sites using ezimage, and those using ngremotemedia.
This way we can have one configuration, and this handler should enable us to work with either of those field types.
See https://github.com/netgen/media-site/pull/69/files#diff-6afaddbbe43dc51cb4eae0e8d585ad314747f36a8aef5dfc6602f50ebd785c29

Reply:
Ah, okay, I guess `Fallback` in the name threw me off. Why not just name it `Image`, since the identifier is also `ngsite/image` ?

Reply:
Done in 8444389725bc90b8c92dc18a33c3b936c4b025e9